pkg/infrastructure/jobs_service.go

```diff
@@ -22,6 +22,19 @@ import (
 )
 
 const (
+	// DefaultJobsServiceImageDomain is the default domain for the Jobs Service image
+	DefaultJobsServiceImageDomain = "quay.io"
+	// DefaultJobsServiceImageNamespace is the default namespace for the Jobs Service image
+	DefaultJobsServiceImageNamespace = "kiegroup"
+	// DefaultJobsServiceImageName is the default image name for the Jobs Service image
+	DefaultJobsServiceImageName = "kogito-jobs-service"
+	// DefaultJobsServiceImageTag is the default image tag for the Jobs Service image
+	DefaultJobsServiceImageTag = "latest"
+	// DefaultJobsServiceImageFullTag is the default full tag name for the Jobs Service image
+	DefaultJobsServiceImageFullTag = DefaultJobsServiceImageDomain + "/" + DefaultJobsServiceImageNamespace + "/" + DefaultJobsServiceImageName + ":" + DefaultJobsServiceImageTag
+	// DefaultJobsServiceName is the default name for the Jobs Services instance service
+	DefaultJobsServiceName = DefaultJobsServiceImageName
```

Comment:
The name of the service should be jobs-service. The CR is already named ‘kogito-Jobs-service’. The same for the Data index, which I would change in the data index improvements jira. Also I’m not sure if moving this variables to this package makes much sense. I will rethink on data index refactoring. We will have a single package for creating services. A lot of code will go away. Thanks for the refactoring. Indeed the smoke should not depend on controller nor cmd. :)

Reply:
ok, we can discuss all of that when you get back :)